shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.mm

```diff
@@ -241,6 +241,8 @@ - (NSMutableArray*)backdropFilterSubviews {
 
 @interface FlutterClippingMaskView ()
 
+@property(nonatomic) CATransform3D reverseScreenScale;
```

Comment:
This needs a declaration comment, per style guide.

Reply:
done